### ISSUE TITLE
Json should be replaced with STON(JSON)

### ISSUE THREAD

**Issue by pavel-krivanek** [pharo-project/pharo-launcher#145]
Launcher currently depends on Json. We should switch it to STONJSON (if not STON directly)

**Comment by demarey** [comment]
I agree. Json was there at the beginning and lack of time to move Json to Ston but only ston should be used.
